packages/react-chain-helmet/src/ReactChainHelmet.ts

```diff
@@ -0,0 +1,10 @@
+import * as Helmet from 'react-helmet'
+
+const helmet = () => (session: any) => {
+  session.on('server', (render: Function) => {
+    render()
+    session.props.helmet = Helmet.rewind()
+  })
```

Comment:
naming is hard: not sure about `on('server', ...)` (should be a verb/action?) and `session.props` (documentProps?).